javascript/javascript1/week1/freecodecamp.js

```diff
@@ -0,0 +1 @@
+const myFreeCodeCamp = 'https://www.freecodecamp.org/Vipavee';
```

Comment:
I can't see that you have finished the sections that has been mentioned in the HW so I'm not sure if you haven't done it or I can't see it

Reply:
Sorry for late reply. I just have updated my timeline to public. Now you can see it :)